x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/utils.test.ts

```diff
@@ -91,3 +102,60 @@ describe('crawlerDataServerToClient', () => {
     expect(output.domains[1]).toEqual(crawlerDomainServerToClient(domains[1]));
   });
 });
+
+describe('crawlDomainValidationToResult', () => {
+  test('with warning', () => {
```

Comment:
[not a blocker] interesting, any particular reason for the switch to `test()` vs `it()` in this file here? I do think the title grammar sounds more natural with the current test() syntax - with it() you'd have to change it to something like `it('handles warnings')` 🤔 up to you if you want to leave it or change for consistency w/ the rest of the codebase

Reply:
oh I think I copied this from a test Orhan wrote 😅

Reply:
I moved these to use `it()` in 746836f

Reply:
looks great, thanks!